doc/safety.md

```diff
@@ -0,0 +1,61 @@
+# SECURITY
+
+This document describes the security recommendations and issues that can be helpful for users of Bitcoin Core.
+
+
+Download Bitcoin Core only from below links, verify the integrity of the download before using:
```

Comment:
```suggestion
Download Bitcoin Core only from the links listed below and verify the integrity of the download before using:
```

Reply:
Done in `088d99b5e5c9bfd907861ce68692e638ee56448d`